thorlcr/msort/tsortmp.cpp

```diff
@@ -77,7 +77,7 @@ bool SortSlaveMP::sendRecv(CMessageBuffer &mb, unsigned timeout)
             mb.read(err);
             StringAttr errstr;
             mb.read(errstr);
-            throw MakeStringException(err, "%s", errstr.get());
+            throw MakeStringException(err, "SortSlaveMP(error from slave %u) - %s", (unsigned)rank, errstr.get());
```

Comment:
It was the addition of slave number that was the significant new piece of information .
I'll replace SortGlobalMP with something else.